lib/coins.py

```diff
@@ -293,6 +295,15 @@ def electrum_header(cls, header, height):
     def deserializer(cls):
         return Deserializer
 
+    @classmethod
+    def daemon(cls):
+        return Daemon
+
+    @classmethod
+    def block_processor(cls):
+        return BlockProcessor
+
```

Comment:
I prefer a class variable but did it in the same way as the:

```
def deserializer(cls):
        return Deserializer
```

If you like, I could refactor the `deserializer()` to be property as well.